docs/case-studies/deploy-to-edge.md

```diff
@@ -0,0 +1,314 @@
+---
+title: Edge Application Delivery
+---
+
+In this section, we will introduce how to deploy applications in edge scenario. We will use some concepts provided by [OpenYurt](https://openyurt.io) to 
+describe what is the key problem we are trying to solve and how KubeVela play a role in this scenario.
+
+:::note
+In this section, we'll use some new features. Make sure your KubeVela version is `>=1.6.0`.
+:::
+
+## Key Problem
+
+The `NodePool` is a CRD defined by OpenYurt. Generally, the `NodePool` is used to describe a group of nodes with the same features.
+A Kubernetes cluster can have multiple `NodePool`s. How to deploy edge applications to different node pools is a key problem in edge scenario.
+
+<details>
+<summary>How OpenYurt define node pool</summary>
+
+> In edge scenario, the edge nodes can usually be grouped by their locations, or other logical characteristics, such as their CPU architecture, telecom carriers or cloud providers.
+The nodes among different groups could also be isolated from each other by various obvious reasons, such as network unreachablility, resource unsharability, and application independence.
+This is the origin of the edge Node Pool.
+> -- [OpenYurt NodePool](https://openyurt.io/docs/core-concepts/yurt-app-manager/#components-introduction)
+ 
+</details>
+
+There are some problems when deploy application to node pools:
+
+1. **Unified Configuration**: We don't want to copy manifests for many times and do some changes for every app in each node pool.
+   Instead, the better choice is use a unified configuration with some parameters to describe the differences.
+2. **Properties differences**: Workloads deploy to different node pools will be similar in common properties.
+   The difference is how to use `NodeSelector`, `Toleration` and `NodeAffinity` to schedule workloads to different node pools.
+3. **Scalability**: The kinds of application is increasing. We don't need to change too much code to support a new kind of application.
+
+See how KubeVela helps to solve these problem.
+
+## Deploy to Edge with KubeVela
+
+We will take a ingress controller example to show how to deploy applications to edge with KubeVela. In this case, we want to deploy a nginx 
+ingress controller to two node pools to implement the traffic in environment with network isolation. Nodes in one node 
+pool will access the ingress in the same node pool and then access the service in the same node pool. 
+
+![edge-ingress-topology](../resources/edge-ingress-topology.png)
```

Comment:
ingress controller is used for exposing internal service out to cluster, so the traffic that access nginx controller should come from outside clients instead of cluster pods.